targets/XPlatCppSdk/source/licenses/curl.txt

```diff
@@ -0,0 +1,12 @@
+https://curl.haxx.se/docs/copyright.html
```

Comment:
Again, this is a file that we should not add unless we make a specific decision to change the way we're building from SdkGenerator.